release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -593,6 +593,11 @@ A Node Tuning Operator (NTO) metric is now added to Telemetry. Follow the proced
 
 The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for examining allocated resources on a worker node. It accounts for resources that are available to be allocated to new pod on a per-zone basis, where a zone can be a Non-Uniform Memory Access (NUMA) node. See xref:../hardware_enablement/psap-node-feature-discovery-operator.adoc#using-the-nfd-topology-updater_node-feature-discovery-operator[Using the NFD Topology Updater] for more information.
 
+
+[id="ocp-4-10-hyperthreading-aware-CPU-manager-policy"]
+==== Hyperthreading-aware CPU manager policy (Technology Preview)
+Hyperthreading-aware CPU manager policy in Openshift Container Platform is now available without the need for extra tuning. The cluster administrator can enable this feature if required. Hyperthreads are abstracted by the hardware as logical processors. Hyperthreading allows a single physical processor to execute two heavyweight threads (processes) at the same time, dynamically sharing processor resources.
```

Comment:
```suggestion
Hyperthreading-aware CPU manager policy in {product-title] is now available without the need for extra tuning. The cluster administrator can enable this feature if required. Hyperthreads are abstracted by the hardware as logical processors. Hyperthreading allows a single physical processor to execute two heavyweight threads (processes) at the same time, dynamically sharing processor resources.
```

Reply:
Are there docs for this that you can link to? Didn't think to ask yesterday, but it would be helpful to add a link if there is documentation.

Reply:
I asked this of the team already, and was told not to link, as the content on the link is not supported by Red Hat.